### PR TITLE
Added designs on the buttons

### LINKED ISSUE #62: [feat] Adding Design on the buttons
### Would you like to have a new feature? Please describe.

Want to add some design on the buttons.

See this video once

https://user-images.githubusercontent.com/70086051/197324529-3d9c75a6-6080-4f18-a33f-370708224f5f.mp4

https://user-images.githubusercontent.com/70086051/197327062-8a9d1c98-1848-4d03-8a4a-17bafa078bbb.mp4

### How would this feature be beneficial for the project

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>

## Fixes Issue #62

This PR fixes the following issues:

closes #62 

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots 
Add all the screenshots which support your changes

### CODE REVIEW

File: src/components/Blog.js [hunk 1]
```diff
@@ -1,25 +1,38 @@
-import React from 'react'
-import '../styles/Blog.css'
-import blog from '../assets/blog.png'
+import React from "react";
```
Comment (AnuragThePathak) [inline src/components/Blog.js:1]: Unnecessary changes and you are not following the code-style in the project. Replace by single quotes and remove the semis.

**Review by AnuragThePathak — CHANGES_REQUESTED** [review]

File: src/components/Blog.js (hunk 1 above)
Comment (AnuragThePathak) [reply src/components/Blog.js:1]: Yeah I understand that, but the problem is that this project is still under work, so there are no style configuration files provided yet. For the time being, maybe you can update your vs code configuration to remove semi-colons in Javascript and prefer single quotes and it should solve the problem.

**Comment by bishal7679** [comment]
@AnuragThePathak Now all good to go!

**Comment by AnuragThePathak** [comment]
Cool, rest of the changes would be reviewed by someone from the dev team.

**Comment by SuperAayush** [comment]
Hey, @bishal7679 can you please quickly resolve the conflicts and commit ASAP 😄?

**Comment by bishal7679** [comment]
@SuperAayush This conflicts happened due to this PR https://github.com/commclassroom/commclassroom/pull/68

**Comment by SuperAayush** [comment]
> @SuperAayush This conflicts happened due to this PR #68

Yes I am aware of that, #68 PR had the final changes for the homepage.
Can you please fetch the data, fix the conflicts and push the changes?

**Comment by bishal7679** [comment]
> > @SuperAayush This conflicts happened due to this PR #68
> 
> Yes I am aware of that, #68 PR had the final changes for the homepage. Can you please fetch the data, fix the conflicts and push the changes?

This final changes looks good! Actually I was facing some line-height issue for the Explore courses button. Now it is good to go!

**Comment by SuperAayush** [comment]
> > > @SuperAayush This conflicts happened due to this PR #68
> > 
> > 
> > Yes I am aware of that, #68 PR had the final changes for the homepage. Can you please fetch the data, fix the conflicts and push the changes?
> 
> This final changes looks good! Actually I was facing some line-height issue for the Explore courses button. Now it is good to go!

Conflicts are yet to be solved!!

**Comment by bishal7679** [comment]
> > > > @SuperAayush This conflicts happened due to this PR #68
> > > 
> > > 
> > > Yes I am aware of that, #68 PR had the final changes for the homepage. Can you please fetch the data, fix the conflicts and push the changes?
> > 
> > 
> > This final changes looks good! Actually I was facing some line-height issue for the Explore courses button. Now it is good to go!
> 
> Conflicts are yet to be solved!!

@SuperAayush sure! I will push it soon!

**Comment by AnuragThePathak** [comment]
@bishal7679 let us know if you need help in conflict resolution.

**Comment by bishal7679** [comment]
@AnuragThePathak @SuperAayush The page is not loading on localhost:3000

**Comment by AnuragThePathak** [comment]
Maybe your site is crashing due to some errors.

**Comment by bhavyastar** [comment]
Can you give the error message ? @bishal7679

**Comment by bishal7679** [comment]
@bhavyastar The page is loading with completely blank
Screenshot (1457)
Screenshot (1458)

**Comment by bishal7679** [comment]
@bhavyastar Now it is loading after run this command 
`npm audit fix --force`

**Comment by bishal7679** [comment]
@bhavyastar @SuperAayush The previous design of the site was looking quite good but in the current one
both side the space is looking so weird and so much blank!
WDYT?
Just giving my point of view :)

**Pull request closed without merging**